ui/packages/ui/src/app/app.tsx

```diff
@@ -1,31 +1,26 @@
 import React from "react";
-import { Page } from "@patternfly/react-core";
 import { BrowserRouter as Router, Route } from "react-router-dom";
 import AppHeader from "./appHeader";
```

Comment:
AppHeader is no longer in used it can be removed. we can add  "no-unused-variable": true in tslint if we want to check it explicitly.